server/graphql/types/src/types/store_preference.rs

```diff
@@ -82,6 +83,18 @@ impl StorePreferenceNode {
 
 impl StorePreferenceNode {
     pub fn from_domain(store_preference: StorePreferenceRow) -> StorePreferenceNode {
-        StorePreferenceNode { store_preference }
+        StorePreferenceNode {
+            store_preference: StorePreferenceRow {
+                monthly_consumption_look_back_period: if store_preference
+                    .monthly_consumption_look_back_period
+                    == 0.0
+                {
+                    DEFAULT_AMC_LOOKBACK_MONTHS.into()
+                } else {
+                    store_preference.monthly_consumption_look_back_period
+                },
+                ..store_preference
+            },
+        }
```

Comment:
@andreievg FYI I added this here because store preferences are called from a couple of places which bypass the service layer, and I liked having this mapping all in one place.

However I think this is relatively unique for from_domain functions, so I wasn't sure if this was the right way to go.